resources/object_repo/developer_catalog_obj.resource

```diff
@@ -25,6 +25,6 @@ ${btnTopologyView}              //button[@data-test-id="topology-switcher-view"
 ${elemTopInstancePan}           //a[contains(.,"<<instance>>")]
 ${txtTopListInstance}           //span[contains(.,"DBaaS")]/parent::div[contains(.,"<<instance>>")]
 ${txtBxDisplayOptions}          //span[contains(.,"Display options")]
-${hdrTopBindInstr}              //h1[contains(.,"Topology View Binding Instructions")]
+${hdrTopBindInstr}              //h1[contains(@id,"pf-modal-part")]/span[./text()="Topology View Binding Instructions"]
 ${btnTopBindClose}              //button[@aria-label="Close"]
-${btnTopBindContinue}           //button[contains(.,"Continue")]
+${btnTopBindContinue}           //button[./text()="Continue" and @aria-disabled="false"]
```

Comment:
@harshkumarRH On Xpath - to find an element containing innerText, We can use either "." or "text()". I am bit confused on "./text()", is this "or" condition? If yes, I hope "." would fulfill the need.

Reply:
I do not have extensive experience with this, I followed the suggestions mentioned here - 

https://stackoverflow.com/questions/7765797/selenium-how-to-locate-a-node-using-exact-text-match
https://bangladroid.wordpress.com/2018/05/24/xpath-how-to-locate-a-node-using-exact-text-match/
https://www.linkedin.com/pulse/xpath-how-locate-element-object-using-exact-text-match-rajendiran

Please let me know if it is best practice to use only "text()=", I'll amend accordingly

Reply:
Changed locator to use only "."
Updated with latest commit